seed/models/data_quality.py

```diff
@@ -696,107 +696,102 @@ def _check(self, rules, row):
                 #                                             len(model_labels['label_ids'])))
 
         for rule in rules:
-            # create an extra data flag for the rule
-            is_extra_data = rule.field in row.extra_data
-
-            # check if the field exists
-            if hasattr(row, rule.field) or is_extra_data:
```

Comment:
CHANGE: `rule.field in row.extra_data` wasn't actually checking if the rule was for an extra_data column, since row.extra_data could just be missing keys. This check was removed, which lead to the removal of the top level `if` statement.